notebooks/ir_scattered_light_calwf3_corrections/Correcting_for_Scattered_Light_in_IR_Exposures_Using_calwf3_to_Mask_Bad_Reads.ipynb

```diff
@@ -0,0 +1,726 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Querying CRDS for the Reference File<a id=\"ref\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before running `calwf3`, we need to set some environment variables.\n",
+    "\n",
+    "We will point to a subdirectory called `crds_cache` using the IREF environment variable, which is used for WFC3 reference files. Other instruments use other variables, e.g., JREF for ACS."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ['CRDS_SERVER_URL'] = 'https://hst-crds.stsci.edu'\n",
+    "os.environ['CRDS_SERVER'] = 'https://hst-crds.stsci.edu'\n",
+    "os.environ['CRDS_PATH'] = './crds_cache'\n",
+    "os.environ['iref'] = './crds_cache/references/hst/wfc3/'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The code block below will query CRDS for the best reference files currently available for our dataset and update the header keywords to point to these new files. We will use the Python package *os* to run terminal commands. In the terminal, the line would be:"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "crds bestrefs --files [filename] --sync-references=1 --update-bestrefs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "...where 'filename' is the name of your fits file.\n",
+    "\n",
+    "\n",
+    "**Warning: this cell may take a few minutes to complete.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "raw_file = f'{file_id}_raw.fits'\n",
+    "\n",
+    "\n",
+    "print( f\"Querying CRDS for the reference file associated with {raw_file}.\")\n",
+    "command_line_input = 'crds bestrefs --files {:} --sync-references=1 --update-bestrefs'.format(raw_file)\n",
+    "os.system(command_line_input)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Reprocessing the Observation<a id=\"main\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As discussed in the [introduction](#intro) to this notebook, the accuracy of the ramp fit performed during the CRCORR cosmic-ray rejection step of the pipeline determines the quality of the calibrated WFC3/IR FLT data products. Given that a time variable background can compromise the quality of the ramp fit, observations affected by TVB will likely result in poor-quality calibrated FLT images ([Appendix: Variable Background Correction of the WFC3/IR 2021-01 ISR](https://www.stsci.edu/files/live/sites/www/files/home/hst/instrumentation/wfc3/documentation/instrument-science-reports-isrs/_documents/2021/2021-01.pdf).)\n",
```

Comment:
from
(Appendix: Variable Background Correction of the WFC3/IR 2021-01 ISR.)
to
(see the Appendix of WFC3 ISR 2021-01).